stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -132,6 +132,28 @@ internal class _AnySequenceBox<Element> {
 
   internal func _underestimateCount() -> Int { _abstract() }
 
+  internal func _map<T>(@noescape transform: Element throws -> T) rethrows -> [T] { _abstract(); }
```

Comment:
Please remove semicolons and wrap to 80 columns (here and everywhere in the patch).